frontend/src/pages/DataCatalog/DataCatalogTab.tsx

```diff
@@ -33,7 +33,6 @@ function DataCatalogTab() {
         indicatorColor="primary"
         textColor="primary"
         centered
-        style={{ marginTop: "40px" }}
```

Comment:
What's the reason for removing this? seems unrelated to the PR title

Reply:
I applied this style to all Tabs in MaterialTheme.tsx so it wasn't needed here any longer